elements/pfe-accordion/package.json

```diff
@@ -8,25 +8,38 @@
   "publishConfig": {
     "access": "public"
   },
-  "description": "Accordion element for PFElements",
+  "description": "Accordion for PatternFly Elements",
+  "keywords": [
+    "web-components",
+    "html"
+  ],
   "repository": {
     "type": "git",
-    "url": ""
+    "url": "https://github.com/patternfly/patternfly-elements/tree/v1.0.0-prerelease.10/elements/pfe-accordion"
   },
-  "main": "index.js",
+  "main": "pfe-accordion.js",
   "scripts": {
     "build": "../../node_modules/.bin/gulp && ../../node_modules/.bin/prettier --ignore-path ../../.prettierignore --write '**/*.{js,json}'",
     "dev": "../../node_modules/.bin/gulp dev",
     "test": "../../node_modules/.bin/wct --configFile ../../wct.conf.json elements/pfe-accordion/test/"
   },
-  "author": {
-    "name": "Kyle Buchanan"
-  },
+  "contributors": [
+    {
+      "name": "Kyle Buchanan",
+      "email": "kylebuch8@gmail.com",
+      "url": "https://github.com/kylebuch8"
+    },
+    {
+      "name": "cd"
+    },
+    {
+      "name": "Kendall Totten"
+    }
+  ],
   "license": "MIT",
   "dependencies": {
     "@patternfly/pfe-sass": "^1.0.0-prerelease.11",
     "@patternfly/pfelement": "^1.0.0-prerelease.11"
   },
-  "generator-pfelement-version": "0.5.5",
```

Comment:
Can we leave the generator version?  It's kind of nice to know what version of the generator was used to generate each element.

Reply:
When we last chatted it seemed like you were wavering on this?  Do you want me to add them back?

Reply:
If not, then this is ready for your review

Reply:
I decided it's best to keep them so I went ahead and added them back in.  Ready to merge!